ztest/ztest.go

```diff
@@ -135,11 +136,30 @@ func Run(t *testing.T, dirname string) {
 // ZTest defines a ztest.
 type ZTest struct {
 	ZQL          string `yaml:"zql"`
-	Input        string `yaml:"input"`
+	Input        Inputs `yaml:"input"`
 	OutputFormat string `yaml:"output-format,omitempty"`
 	Output       string `yaml:"output"`
 }
 
+// Inputs is an array of strings. Its only purpose is to support parsing of
+// both single string and array yaml values for the field ZTest.Input.
+type Inputs []string
+
+func (i *Inputs) UnmarshalYAML(value *yaml.Node) error {
+	if value.Kind == yaml.SequenceNode {
+		var inputs []string
+		err := value.Decode(&inputs)
+		*i = inputs
+		return err
+	}
+	var input string
+	if err := value.Decode(&input); err != nil {
+		return err
+	}
+	*i = append(*i, input)
+	return nil
```

Comment:
Either approach to error handling here is fine, but let's pick one for consistency.

Reply:
If I followed the same approach below as I do on top, an error would decode to an array with a single empty string value. I don't it should do that.

Reply:
👍